calico_kubernetes/tests/kube_plugin_test.py

```diff
@@ -1224,6 +1224,15 @@ def test_configure_logger(self, m_log_to_stdout, m_logging_f, m_logging_sh,
         else:
             m_log.addHandler.assert_called_once_with(f_handler)
 
+    def test_filter(self):
+        """Test filter method for IdentityFilterClass
```

Comment:
If the docstring is only a single line, you can keep the ending quotes on the same line.